### PR TITLE
Add EM_KVX elfid

### PR DESCRIPTION
**Pull request by jmaselbas**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Description**
Add the ELF ID for the KVX architecture familly with the id 256.

**Test plan**
rizin should recognize the elfid in the file https://github.com/rizinorg/rizin-testbins/blob/master/kvx/hello.kv3

**Closing issues**
None

### CODE REVIEW

**Review by XVilka — APPROVED** [review]

**Comment by XVilka** [comment]
Please rebase to resolve the conflict

File: librz/bin/format/elf/elf_info.c [hunk 1]
```diff
@@ -282,6 +283,7 @@ static const struct arch_translation arch_translation_table[] = {
 	{ EM_X86_64, "x86" },
 	{ EM_NONE, "null" },
 	{ EM_BPF, "bpf" }
```
Comment (XVilka) [inline librz/bin/format/elf/elf_info.c]: Missing comma.

**Review by XVilka — CHANGES_REQUESTED** [review]

File: librz/bin/format/elf/elf_info.c (hunk 1 above)
Comment (jmaselbas) [reply librz/bin/format/elf/elf_info.c]: yeah.... I've noticed, is there is reason to not leave a trailing comma ?

Comment (XVilka) [reply librz/bin/format/elf/elf_info.c]: No reason, I think.

**Comment by jmaselbas** [comment]
this time it should be good, I've added missing commas

**Review by ret2libc — APPROVED** [review]

**Pull request merged**